supernode/configs/node.go

```diff
@@ -10,12 +11,14 @@ type Node struct {
 	// `squash` field cannot be pointer
 	ArtworkRegister artworkregister.Config `mapstructure:",squash" json:"artwork_register,omitempty"`
 	Server          *server.Config         `mapstructure:"server" json:"server,omitempty"`
+	MetaDB          *metadb.Config         `mapstructure:"metadb" json:"metadb,omitempty"`
```

Comment:
I think it's better to move it to a higher level in the `Config` structure

Reply:
Agree

Reply:
done